package.json

```diff
@@ -38,14 +38,14 @@
     "@typescript-eslint/parser": "^5.4.0",
     "eslint": "^8.2.0",
     "eslint-config-3box": "^0.4.0",
-    "ethers": "^5.5.1",
     "jest": "^27.3.1",
     "jest-resolver-enhanced": "^1.0.1",
     "prettier": "^2.4.1",
     "regenerator-runtime": "^0.13.9",
     "typescript": "^4.4.4"
   },
   "dependencies": {
+    "@ethersproject/wallet": "^5.5.0",
```

Comment:
I hate that we need this just to verify one secp256k1 signature.

Reply:
We target ES2020 now so it should be treeshaken decently I believe. And still better than using the entire `ethers` I guess that we would have had if we just used the SIWE library.

Reply:
yeah, it's fine. It's just frustrating to know that it could be done much better :)

Reply:
Yeah I was hoping for a @ethersproject/utils library but woops